requirements.txt

```diff
@@ -6,4 +6,4 @@ autopep8
 matplotlib~=3.0
 
 nbformat
-nbconvert==5.6.1
+nbconvert
```

Comment:
We might as well go back to implicit include, as nbconvert was introduced in cf551a8f88d566171aaf291c5210624ff51565a3 only to pin it (and fix a py38 issue). Assuming that works in all environments/pythons.

Reply:
Done!